website/content/docs/connect/config-entries/mesh.mdx

```diff
@@ -156,8 +154,6 @@ The `mesh` configuration entry can only be created in the `default` namespace an
 
 ```hcl
 Kind      = "mesh"
-Namespace = "default" # Can only be set to "default".
-Partition = "default"
```

Comment:
I got feedback from one of the engineers this was confusing, because not specifying anything will get you the right behavior for both OSS/ENT. Wanted to get your thoughts. That might also mean we don't need separate tabs for OSS and ENT.